File: unit1/functions_lab/functions0.c
#include <cs50.h>
#include <stdio.h>

float getTemp();
float convertToFahr(float);
void displayResults(float, float);

int main(void)
{
    float tempC = getTemp();
    float tempF = convertToFahr(tempC);
    displayResults(tempC, tempF);
}

float getTemp()
{
    float tempC = get_float("Enter temp in Celsius: ");
    return tempC;
}

float convertToFahr(float c)
{
    return c * 9 / 5 + 32;
}

void displayResults(float c, float f)
{
    printf("%.2f C = %.2f F\n", c, f);
}

File: unit3/functions0.py
def get_temp():
    temp_c = float(input("Enter temp in Celsius: "))
    return temp_c

def convert_to_fahr(c):
    return c * 9 / 5 + 32

def display_results(c, f):
    print(f"{c:.2f} C = {f:.2f} F")

def main():
    temp_c = get_temp()
    temp_f = convert_to_fahr(temp_c)
    display_results(temp_c, temp_f)

main()